e2e/libs/keywords/sharemanager_keywords.py

```
@@ -67,12 +67,32 @@ def delete_sharemanager_pod_and_wait_for_recreation(self, name):
 
+    def wait_for_sharemanager_pod_restart(self, name):
+        sharemanager_pod_name = "share-manager-" + name
+        sharemanager_pod = get_pod(sharemanager_pod_name, "longhorn-system")
+        last_creation_time = sharemanager_pod.metadata.creation_timestamp
+
+        retry_count, retry_interval = get_retry_count_and_interval()
+        for i in range(retry_count):
+            logging(f"Waiting for sharemanager for volume {name} restart ... ({i})")
+            time.sleep(retry_interval)
+            sharemanager_pod = get_pod(sharemanager_pod_name, "longhorn-system")
+            if sharemanager_pod == None:
```

Ruff (0.8.0)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)